cli.js

```diff
@@ -56,6 +56,11 @@ const cli = meow(`
 
 updateNotifier({pkg: cli.pkg}).notify();
 
+process.on('SIGINT', () => {
+	console.log('\nAborted!');
```

Comment:
Think we should log something else, not sure what though :).

Reply:
You could be specific and write something like "sigint received. exitting."